RELEASE/scripts/autoscend.ash

```diff
@@ -2409,170 +2419,114 @@ boolean LX_attemptPowerLevel()
 {
 	set_property("auto_powerLevelAdvCount", get_property("auto_powerLevelAdvCount").to_int() + 1);
 	set_property("auto_powerLevelLastAttempted", my_turncount());
-
+	set_property("auto_powerLevelLastLevel", my_level());
+	set_property("auto_newbieOverride", true);
+	
 	handleFamiliar("stat");
 	addToMaximize("100 exp");
-
-	if(LX_freeCombats(true))
-	{
-		return true;
-	}
-
-	if(!hasTorso())
+	
+	auto_log_warning("I need to powerlevel", "red");
+	int delay = get_property("auto_powerLevelTimer").to_int();
+	if(delay == 0)
 	{
-		// We need to acquire a letter from Melvign...
-		if(LX_melvignShirt())
-		{
-			return true;
-		}
+		delay = 10;
 	}
+	wait(delay);
 
-	if(auto_my_path() == "The Source")
+	if(LX_freeCombats(true)) return true;
+	
+	if(chateaumantegna_available() && haveFreeRestAvailable() && auto_my_path() != "The Source")
 	{
-		if (get_property("lastSecondFloorUnlock").to_int() == my_ascensions())
-		{
-			if(get_property("barrelShrineUnlocked").to_boolean())
-			{
-				if(cloversAvailable() == 0)
-				{
-					handleBarrelFullOfBarrels(false);
-					string temp = visit_url("barrel.php");
-					temp = visit_url("choice.php?whichchoice=1099&pwd=&option=2");
-					handleBarrelFullOfBarrels(false);
-					return true;
-				}
-				stat myStat = my_primestat();
-				if(my_basestat(myStat) >= 148)
-				{
-					return false;
-				}
-				else if(my_basestat(myStat) >= 125)
-				{
-					//Should probably prefer to check what equipment failures we may be having.
-					if((my_basestat($stat[Muscle]) < my_basestat(myStat)) && (my_basestat($stat[Muscle]) < 70))
-					{
-						myStat = $stat[Muscle];
-					}
-					if((my_basestat($stat[Mysticality]) < my_basestat(myStat)) && (my_basestat($stat[Mysticality]) < 70))
-					{
-						myStat = $stat[Mysticality];
-					}
-					if((my_basestat($stat[Moxie]) < my_basestat(myStat)) && (my_basestat($stat[Moxie]) < 70))
-					{
-						myStat = $stat[Moxie];
-					}
-				}
-				//Else, default to mainstat.
-
-				//Determine where to go for clover stats, do not worry about clover failures
-				location whereTo = $location[none];
-				switch(myStat)
-				{
-				case $stat[Muscle]:			whereTo = $location[The Haunted Gallery];				break;
-				case $stat[Mysticality]:	whereTo = $location[The Haunted Bathroom];				break;
-				case $stat[Moxie]:			whereTo = $location[The Haunted Ballroom];				break;
-				}
-
-				if (whereTo == $location[The Haunted Ballroom] && internalQuestStatus("questM21Dance") > 3)
-				{
-					use(item_amount($item[ten-leaf clover]), $item[ten-leaf clover]);
-					LX_spookyBedroomCombat();
-					return true;
-				}
-				if(cloversAvailable() > 0)
-				{
-					cloverUsageInit();
-				}
-				autoAdv(1, whereTo);
-				cloverUsageFinish();
-				return true;
-			}
-			//Banish mahogant, elegant after gown only. (Harold\'s Bell?)
-			LX_spookyBedroomCombat();
-			return true;
-		}
+		doFreeRest();
+		cli_execute("scripts/autoscend/auto_post_adv.ash");
+		loopHandlerDelayAll();
+		return true;
```

Comment:
I assume this was moved in from `L13_powerlevel()`?

Reply:
Yes. those are lines from L13_powerlevel() which it would do before it called on LX_attemptPowerLevel(). so i moved those parts over to LX_attemptPowerLevel()